profiles/templates/profiles/profile.html

```diff
@@ -6,4 +6,23 @@
     <!-- TO DO : build profile page -->
     TEST PROFILE PAGE
 
+<!DOCTYPE html>
+    <h1>Profile Page</h1>
+    <h2>You are logged in as</h2>
+
+<!---------------------------------------profile display------------------------------------------------->    
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-4"><h4>Username :</h4><p> {{ user.username }} </p></div>
+        <div class="col-md-4"><h4>User e-mail address :</h4><p> {{ user.email }} </p></div>
+        </div>
+        <div class="row">
+        <div class="col-md-4"><h4>First name :</h4><p> {{ user.first_name }}</p></div>
+        <div class="col-md-4"><h4>Last name :</h4><p> {{ user.last_name }}</p></div>
+      </div>
+    </div>
 {% endblock %}
+
+

```

Comment:
Files should generally only have a single empty line at the end of the file.
Can you remember the extra blank lines here, so there's only 1 blank line at the end of the file?

Reply:
Deleted unnecessary empty lines.